hardware/ethernet/enc28j60_process.c

```diff
@@ -57,7 +57,7 @@ void network_process(void)
      * buffer, return */
     if ( !interrupt_occured()
 #   ifdef ENC28J60_REV4_WORKAROUND
-                || pktcnt == 0
+                && pktcnt == 0
```

Comment:
After reading the errata again, I come to the conclusion that there is neither an OR nor an AND right here. The combination with pktcnt brings no added value. Thus the complete block ```#ifdef ENC28J60_REV4_WORKAROUND``` can be deleted.

Further down in the source code the variable pktcnt is used without ```#ifdef ENC28J60_REV4_WORKAROUND```. This should be corrected in the PR, even if it does not currently lead to a compiler error.

The block commented out with _"not needed anymore"_ should also be removed, because since the correction for Rev.6(7) it is no longer needed.